src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -76,6 +85,10 @@ export const WorkspaceForm = ({
 
   const [name, setName] = useState(defaultValues?.name);
   const [description, setDescription] = useState(defaultValues?.description);
+  const [color, setColor] = useState(defaultValues?.color);
+  const [icon, setIcon] = useState(defaultValues?.icon);
+  const [defaultVISTheme, setDefaultVISTheme] = useState(defaultValues?.defaultVISTheme);
```

Comment:
I do not know if it would be good to use a object state instead of multiple isolated states in this case.

Reply:
For now, we don't know too much relations between these three fields. The defaultVISTheme and icon will be little bit complex. The icon may need to support custom icon upload and defaultVISTheme may contains multiple colors. Not sure the final design for this part.